src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\relaxed\Normalizer\EntityReferenceItemNormalizer.
+ */
+
+namespace Drupal\relaxed\Normalizer;
+
+use Drupal\Core\Entity\FieldableEntityStorageInterface;
+use Drupal\serialization\Normalizer\NormalizerBase;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class EntityReferenceItemNormalizer extends NormalizerBase implements DenormalizerInterface {
+
+  /**
+   * The interface or class that this Normalizer supports.
+   *
+   * @var string
+   */
+  protected $supportedInterfaceOrClass = 'Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem';
+
+  /**
+   * @var string[]
+   */
+  protected $format = array('json');
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize($field, $format = NULL, array $context = array()) {
+    $value = $field->getValue();
+    $target_type = $field->getFieldDefinition()->getSetting('target_type');
+    $storage = \Drupal::entityManager()->getStorage($target_type);
+
+    if ($target_type == 'user' || !($storage instanceof FieldableEntityStorageInterface)) {
+      return $value;
+    }
```

Comment:
@jeqq Please add a comment here to explain why we need a special condition for user entity type here.